jacodb-ets/src/main/kotlin/org/jacodb/ets/base/Ops.kt

```diff
@@ -16,260 +16,41 @@
 
 package org.jacodb.ets.base
 
-enum class UnaryOp {
-    /**
-     * '-'
-     */
-    Minus,
-
-    /**
-     * '+'
-     */
-    Plus,
-
-    /**
-     * '!'
-     */
-    Bang,
-
-    /**
-     * '~'
-     */
-    Tilde,
-
-    /**
-     * 'typeof'
-     */
-    Typeof,
-
-    /**
-     * 'void'
-     */
-    Void,
-
-    /**
-     * 'delete'
-     */
-    Delete,
-}
-
-enum class UpdateOp {
-    /**
-     * '++'
-     */
-    Inc,
-
-    /**
-     * '--'
-     */
-    Dec,
-}
-
-enum class BinaryOp {
-    /**
-     * `==`
-     */
-    EqEq,
-
-    /**
-     * `!=`
-     */
-    NotEq,
-
-    /**
-     * `===`
-     */
-    EqEqEq,
-
-    /**
-     * `!==`
-     */
-    NotEqEq,
-
-    /**
-     * `<`
-     */
-    Lt,
-
-    /**
-     * `<=`
-     */
-    LtEq,
-
-    /**
-     * `>`
-     */
-    Gt,
-
-    /**
-     * `>=`
-     */
-    GtEq,
-
-    /**
-     * `<<`
-     */
-    LShift,
-
-    /**
-     * `>>`
-     */
-    RShift,
-
-    /**
-     * `>>>`
-     */
-    ZeroFillRShift,
-
-    /**
-     * `+`
-     */
-    Add,
-
-    /**
-     * `-`
-     */
-    Sub,
-
-    /**
-     * `*`
-     */
-    Mul,
-
-    /**
-     * '/'
-     */
-    Div,
-
-    /**
-     * `%`
-     */
-    Mod,
-
-    /**
-     * `|`
-     */
-    BitOr,
-
-    /**
-     * `^`
-     */
-    BitXor,
-
-    /**
-     * `&`
-     */
-    BitAnd,
-
-    /**
-     * `||`
-     */
-    LogicalOr,
-
-    /**
-     * `&&`
-     */
-    LogicalAnd,
-
-    /**
-     * `in`
-     */
-    In,
-
-    /**
-     * `instanceof`
-     */
-    InstanceOf,
-
-    /**
-     * `**`
-     */
-    Exp,
-
-    /**
-     * `??`
-     */
-    NullishCoalescing,
-}
-
-enum class AssignOp {
-    /**
-     * '='
-     */
-    Assign,
-
-    /**
-     * '+='
-     */
-    AddAssign,
-
-    /**
-     * '-='
-     */
-    SubAssign,
-
-    /**
-     * '*='
-     */
-    MulAssign,
-
-    /**
-     * '/='
-     */
-    DivAssign,
-
-    /**
-     * '%='
-     */
-    ModAssign,
-
-    /**
-     * '<<='
-     */
-    LShiftAssign,
-
-    /**
-     * '>>='
-     */
-    RShiftAssign,
-
-    /**
-     * '>>>='
-     */
-    ZeroFillRShiftAssign,
-
-    /**
-     * '|='
-     */
-    BitOrAssign,
-
-    /**
-     * '^='
-     */
-    BitXorAssign,
-
-    /**
-     * '&='
-     */
-    BitAndAssign,
-
-    /**
-     * '**='
-     */
-    ExpAssign,
-
-    /**
-     * '&&='
-     */
-    AndAssign,
-
-    /**
-     * '||='
-     */
-    OrAssign,
-
-    /**
-     * '??='
-     */
-    NullishAssign,
+object Ops {
+    const val MINUS = "-"
+    const val PLUS = "+"
+    const val BANG = "!"
+    const val TILDE = "~"
+    const val PLUS_PLUS = "++"
+    const val MINUS_MINUS = "--"
+    const val EQ_EQ = "=="
+    const val NOT_EQ_EQ = "!="
```

Comment:
why do we need this additional EQ?

Reply:
That's typo. I am gonna fix this and several other "bugs" you have not noticed later today.